app/src/main/res/values/colors.xml

```diff
@@ -135,4 +135,16 @@
   <color name="seekbarDivider">#707070</color>
   <color name="topicThumbnailBackground">#C6DCDA</color>
   <color name="topicTitleBackground">#2F6687</color>
+  <!-- STATUS BAR COLORS -->
+  <color name="profile_status_bar">#000000</color>
+  <color name="hint_full_dialog_status_bar">#00236E</color>
+  <color name="slide_drawer_open_status_bar">#001F2A</color>
+  <color name="concept_card_status_bar">#882814</color>
+  <color name="pin_input_status_bar">#BDCCCC</color>
+  <color name="walkthrough_status_bar">#CCCCCC</color>
+  <color name="onboarding_1_status_bar">#00847A</color>
+  <color name="onboarding_2_status_bar">#8B2A15</color>
+  <color name="onboarding_3_status_bar">#AD8500</color>
+  <color name="onboarding_4_status_bar">#3B5893</color>
+  <color name="registration_status_bar">#9CB1BE</color>
```

Comment:
Rename all the colors introduced by you in camelCase format:
example:
`onboarding1StatusBar `

Reply:
Done